src/test/scala/edu/ie3/simona/agent/grid/DBFSAlgorithmSupGridSpec.scala

```diff
@@ -334,24 +370,34 @@ class DBFSAlgorithmSupGridSpec
                     )
                   )
                 ) =>
-            case FinishGridSimulationTrigger(3600) =>
-              // when we received a FinishGridSimulationTrigger (as inferior grid agent)
-              // we expect another completion message then as well (scheduler view)
-              expectMsgPF() {
-                case CompletionMessage(
-                      _,
-                      Some(
-                        Vector(
-                          ScheduleTriggerMessage(ActivityStartTrigger(7200), _)
-                        )
-                      )
-                    ) =>
-                // after doing cleanup stuff, our agent should go back to idle again
-                case x =>
-                  fail(
-                    s"Invalid message received when expecting a completion message for simulate grid after cleanup! Message was $x"
+            // when we received a FinishGridSimulationTrigger (as inferior grid agent)
+            // we expect another completion message then as well (scheduler view)
+            case CompletionMessage(
+                  _,
+                  Some(
+                    Vector(
+                      ScheduleTriggerMessage(ActivityStartTrigger(7200), _)
+                    )
                   )
+                ) =>
+              // after doing cleanup stuff, our agent should go back to idle again
+              resultListener.expectMsgPF() {
+                case powerFlowResultEvent: PowerFlowResultEvent =>
+                  powerFlowResultEvent.nodeResults.headOption match {
+                    case Some(value) =>
+                      value.getvMag().getValue shouldBe 1
+                      value.getvAng().getValue shouldBe 0
+                  }
+
+                  powerFlowResultEvent.lineResults shouldBe empty
+                  powerFlowResultEvent.switchResults shouldBe empty
+                  powerFlowResultEvent.transformer2wResults shouldBe empty
+                  powerFlowResultEvent.transformer3wResults shouldBe empty
```

Comment:
Maybe it's worthwhile to adapt the `simonaConfig` (or rather, adapt a copy of it) so that power flow grid results are sent. `simona.output.grid` should be the right place